README.md

```diff
@@ -9,4 +9,5 @@
 1. [Testing](/testing)
 1. [Authentication Server in C#](/c-sharp-server)
 1. [Change Data Capture](https://github.com/getditto/external-sync): Keep your Ditto database in sync with external systems, such as MongoDB.
+1. [Heartbeat](https://github.com/getditto/heartbeat): Create an end-to-end heartbeat to monitor Big Peer status 
```

Comment:
This URL automatically redirects to https://github.com/getditto/template-heartbeat instead, which I believe is fine.